staging/src/k8s.io/apimachinery/pkg/util/validation/field/error_matcher.go

```diff
@@ -14,37 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package testing
+package field
 
 import (
 	"fmt"
 	"reflect"
 	"regexp"
 	"strings"
-	"testing"
-
-	field "k8s.io/apimachinery/pkg/util/validation/field"
 )
 
-// ErrorMatcher is a helper for comparing field.Error objects.
+// ErrorMatcher is a helper for comparing field Error objects.
 type ErrorMatcher struct {
 	// TODO(thockin): consider whether type is ever NOT required, maybe just
 	// assume it.
 	matchType bool
 	// TODO(thockin): consider whether field could be assumed - if the
-	// "want" error has a nil field, don't match on field.
+	// "want" error has a nil field, don't match on
```

Comment:
Looks like an automated search and replace went wrong :)

Reply:
fixed, thanks